pirania/files/usr/libexec/rpcd/pirania

```diff
@@ -64,6 +70,56 @@ local function add_many_vouchers(msg)
     printJson(result)
 end
 
+local function add_visitor_voucher(msg)
+    local result = {}
```

Comment:
Also the files in rpcd should have the minimum code posible, and most of the code live in "library files"